docs/quick-start.md

```diff
@@ -0,0 +1,94 @@
+	- [You can read more about using the CLI here](https://github.com/Automattic/jetpack/blob/master/tools/cli/README.md).
+
+## Running Jetpack locally
+
+After everything is installed, you're ready to run Jetpack locally! While there are other supported methods of doing this, we recommend and support using Docker containers. 
+
+To setup Docker:
+- Install Docker:
+	- Mac: `brew install --cask docker`
+	- Linux: `brew install docker` 
+- Copy the settings file and modify the site password and any other configuration you'd like: `cp tools/docker/default.env tools/docker/.env`
+- Start the Docker container using `jetpack docker up`
+- Install WordPress in your Docker container using `jetpack docker install` 
+- Open up http://localhost to see your site!
+
+For more in depth Docker instructions, follow the [Docker environment for Jetpack Development guide](../tools/docker/README.md).
+
+### Setting up Jurassic Tube
+
+To test network related features of Jetpack, you'll need a test site that can create local HTTP tunnels. If you're an Automattician, we recommend using Jurassic Tube:
+
+- Add a subdomain on [jurassic.tube](https://jurassic.tube/)
+- Make sure you've run `pnpm install && pnpm cli-link`
+- Make sure Docker is running `jetpack docker up -d`
+- Stand on the monorepo root in your terminal and run `mkdir tools/docker/bin/jt`
+- Download and run the installation script: `curl "https://jurassic.tube/get-installer.php?env=jetpack" -o tools/docker/bin/jt/installer.sh && chmod +x tools/docker/bin/jt/installer.sh && tools/docker/bin/jt/installer.sh`
+- Set your username: `jetpack docker jt-config username [your-username-here e.g david]`
+- Set your subdomain: `jetpack docker jt-config subdomain [your-subdomain-here e.g. spaceman]` 
+- Now, you can start your site with `jetpack docker jt-up`
+- Your site should be avalable at `https://custom-subdomain.jurassic.tube`
+
+Note: This is for Automattician use only. For other methods, check out [ngrok](https://github.com/Automattic/jetpack/blob/master/tools/docker/README.md#using-ngrok-with-jetpack) or [another similar service](https://alternativeto.net/software/ngrok/).
+
+### Check if your environment is ready for Jetpack development
+
+We provide a script to help you in assessing if everything's ready on your system to contribute to Jetpack.
+
+```sh
+tools/check-development-environment.sh
+```
+
+Running the script will tell you if you have your environment already set up and what you need to do in order to get it ready for Jetpack development:
+
+- All green `YES` or `OK` messages mean you're ready to start
+- Red `NO` messages mean something is wrong or missing, and a link will be provided to help you with a fix.
+- Yellow messages indicate something optional is broken or missing.
+
+#### Running Tests
+
+To run tests, you can use the Jetpack CLI: `jetpack test` and then choose the project and type of test you'd like to run.
```

Comment:
This seems a bit out of place here; I wasn't sure what tests we were talking about. Is this to test that my development environment is ready to be used, since that's what we were just talking about?

At this point I don't know if I would expect a mention about tests. I would personally want to be sure my local site is working at this point.

Maybe tests should be moved to a dedicated section?

Reply:
That was a stopgap section to address some concerns from above: https://github.com/Automattic/jetpack/pull/22074#discussion_r768146874

I added some more information to make things clearer. Not committed to the idea of having it here and am open to further discussion as we evolve this.